packages/sdk/src/domain/constants/common.ts

```diff
@@ -0,0 +1 @@
+export const ASSET_URL = "https://helio-assets.s3.eu-west-1.amazonaws.com";
```

Comment:
do we need this hardcoded here? if we migrate the s3 to a different package account this might cause problems

Reply:
do you think we can get it from BE?